<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" ignored="14" total="22" passed="6">
  <reporter-output>
  </reporter-output>
  <suite name="Adactin Hotel App Default test suite" duration-ms="60533" started-at="2019-08-16T00:25:08Z" finished-at="2019-08-16T00:26:08Z">
    <groups>
    </groups>
    <test name="Smoke tests" duration-ms="60533" started-at="2019-08-16T00:25:08Z" finished-at="2019-08-16T00:26:08Z">
      <class name="MAVENModule._SeleniumAdactinTests.TC_101">
        <test-method status="PASS" signature="createWebDriver(java.lang.String)[pri:0, instance:MAVENModule._SeleniumAdactinTests.TC_101@3c19aaa5]" name="createWebDriver" is-config="true" duration-ms="3497" started-at="2019-08-16T00:25:08Z" finished-at="2019-08-16T00:25:11Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[CHROME]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createWebDriver -->
        <test-method status="PASS" signature="tc101_verify_valid_login()[pri:0, instance:MAVENModule._SeleniumAdactinTests.TC_101@3c19aaa5]" name="tc101_verify_valid_login" duration-ms="8995" started-at="2019-08-16T00:25:11Z" finished-at="2019-08-16T00:25:20Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tc101_verify_valid_login -->
        <test-method status="PASS" signature="releaseWebDriver()[pri:0, instance:MAVENModule._SeleniumAdactinTests.TC_101@3c19aaa5]" name="releaseWebDriver" is-config="true" duration-ms="1013" started-at="2019-08-16T00:26:07Z" finished-at="2019-08-16T00:26:08Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- releaseWebDriver -->
      </class> <!-- MAVENModule._SeleniumAdactinTests.TC_101 -->
      <class name="MAVENModule._SeleniumAdactinTests.TC_102">
        <test-method status="PASS" signature="tc102_verify_date_error_msgs()[pri:0, instance:MAVENModule._SeleniumAdactinTests.TC_102@3349e9bb]" name="tc102_verify_date_error_msgs" duration-ms="4869" started-at="2019-08-16T00:25:20Z" finished-at="2019-08-16T00:25:25Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tc102_verify_date_error_msgs -->
      </class> <!-- MAVENModule._SeleniumAdactinTests.TC_102 -->
      <class name="MAVENModule._SeleniumAdactinTests.TC_103">
        <test-method status="FAIL" signature="tc103()[pri:0, instance:MAVENModule._SeleniumAdactinTests.TC_103@409bf450]" name="tc103" duration-ms="19936" started-at="2019-08-16T00:25:25Z" finished-at="2019-08-16T00:25:45Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.fail(Assert.java:103)
at MAVENModule._SeleniumAdactinTests.TC_103.tc103(TC_103.java:125)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tc103 -->
      </class> <!-- MAVENModule._SeleniumAdactinTests.TC_103 -->
      <class name="MAVENModule._SeleniumAdactinTests.TC_108">
        <test-method status="FAIL" signature="test08()[pri:0, instance:MAVENModule._SeleniumAdactinTests.TC_108@55b699ef]" name="test08" duration-ms="4313" started-at="2019-08-16T00:26:03Z" finished-at="2019-08-16T00:26:07Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [250.0] but found [650.0]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [250.0] but found [650.0]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at MAVENModule._SeleniumAdactinTests.TC_108.test08(TC_108.java:85)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test08 -->
      </class> <!-- MAVENModule._SeleniumAdactinTests.TC_108 -->
      <class name="MAVENModule._SeleniumAdactinTests.TC_104">
        <test-method status="PASS" signature="test04()[pri:0, instance:MAVENModule._SeleniumAdactinTests.TC_104@38d8f54a]" name="test04" duration-ms="5470" started-at="2019-08-16T00:25:45Z" finished-at="2019-08-16T00:25:51Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test04 -->
      </class> <!-- MAVENModule._SeleniumAdactinTests.TC_104 -->
      <class name="MAVENModule._SeleniumAdactinTests.TC_105">
        <test-method status="PASS" signature="tc105()[pri:0, instance:MAVENModule._SeleniumAdactinTests.TC_105@49e53c76]" name="tc105" duration-ms="4175" started-at="2019-08-16T00:25:51Z" finished-at="2019-08-16T00:25:55Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tc105 -->
      </class> <!-- MAVENModule._SeleniumAdactinTests.TC_105 -->
      <class name="MAVENModule._SeleniumAdactinTests.TC_106">
        <test-method status="PASS" signature="tc106()[pri:0, instance:MAVENModule._SeleniumAdactinTests.TC_106@351d00c0]" name="tc106" duration-ms="4231" started-at="2019-08-16T00:25:55Z" finished-at="2019-08-16T00:25:59Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tc106 -->
      </class> <!-- MAVENModule._SeleniumAdactinTests.TC_106 -->
      <class name="MAVENModule._SeleniumAdactinTests.TC_107">
        <test-method status="PASS" signature="test07()[pri:0, instance:MAVENModule._SeleniumAdactinTests.TC_107@2a3b5b47]" name="test07" duration-ms="3979" started-at="2019-08-16T00:25:59Z" finished-at="2019-08-16T00:26:03Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test07 -->
      </class> <!-- MAVENModule._SeleniumAdactinTests.TC_107 -->
    </test> <!-- Smoke tests -->
  </suite> <!-- Adactin Hotel App Default test suite -->
</testng-results>
